### DISCUSSION TITLE
Why does it ask for a username and password if I already logged in with my token?

### DISCUSSION THREAD

**Discussion by Portu87** [community/community#60335] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Why is it asking for a username and password if I already logged in with my token? I'm using Ubuntu 22.04 and have "gh" installed correctly. I successfully accessed my account, but when I run "gh repo clone", it asks for a username and password. And it doesn't let me enter the password.
`ubuntu@ip-***-**-**-**:~$ gh auth status
github.com
  ✓ Logged in to github.com as Portu87 (/home/ubuntu/.config/gh/hosts.yml)
  ✓ Git operations for github.com configured to use https protocol.
  ✓ Token: *******************
  
ubuntu@ip-***-**-**-**:~$ gh repo clone Portu87/server-foodyapp
Cloning into 'server-foodyapp'...
Username for 'https://github.com': Portu87
Password for 'https://Portu87@github.com': 
ubuntu@ip-***-**-**-**~$ gh auth status
github.com
  ✓ Logged in to github.com as Portu87 (/home/ubuntu/.config/gh/hosts.yml)
  ✓ Git operations for github.com configured to use https protocol.
  ✓ Token: *******************
  
ubuntu@ip-***-**-**-**:~$ `

**Comment by nullsecX270** [comment]
safety induction
